docs/modules/ROOT/pages/subscriptions/getting-started.adoc

```diff
@@ -105,6 +94,8 @@ async function main() {
 main();
 ```
 
+NOTE: The example above uses the **xref::subscriptions/plugins/single-instance.adoc[single instance plugin]** and is not intended for production.
```

Comment:
I really don't think we should say this, it would be fine in production for a system which doesn't need to horizontally scale.

Reply:
```suggestion
NOTE: The example above uses the **xref::subscriptions/plugins/single-instance.adoc[single instance plugin]** and cannot scale horizontally.
```